packages/Button/stories/variations/LinkButtonVariations.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { Rule } from "storybook/assets/styles/common.styles";
+import PlusIcon from "@paprika/icon/lib/Add";
+import Button from "../../src/Button";
+
+function clickHandler() {
+  action("Clicked a button")();
+}
+
+export default function LinkButtonVariations() {
+  return (
+    <>
+      <p>
```

Comment:
Let's add a "Variations of <Button.Link>" heading here.  And for the others. You get the idea.